examples/resources/humanitec_agent/import.sh

```diff
@@ -0,0 +1 @@
+terraform import humanitec_agent.example agent_id
```

Comment:
I have some concern here.
The problem is that we need a public key to create the agent, but then when we fetch the agents by id, we don't have any public key (we could fetch the keys registered for an agent but we can have multiple of them).

So when we import the agent, it's not clear to me how we should also link a key to it. As we need to do it as `public_key` is required in the `agent` resource.
If we don't make the `public_key` required, we would avoid to have the breaking resource when we import an `agent` but that would break to create an agent without public key, when the API will be hit.

Reply:
An option might be to make the keys part of the `humanitec_agent` resource but it seems more tricky to handle with these nested resources.

Reply:
I wonder if we can consider to force the import of a key resource every time we import an agent resource.

Reply:
Maybe I have an idea: we can ask to specify both agent id and key fingerprint to import an agent:

```
terraform import humanitec_agent.my_agent agent_id/key_fingerprint
```

would import a resource of kind `humanitec_agent` storing as public_key in the state the key with fingerprint.

```
terraform import humanitec_agent_key.my_key agent_id/key_fingerprint
```

would import a resource of kind `humanitec_agent_key` .